library/libosbuild/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "libosbuild"
+version = "73.0.0"
+edition = "2021"
+authors = ["supakeen <supakeen@redhat.com>"]
```

Comment:
In newer code I tend to omit `authors` because it becomes out of date quickly - the set of authors is logged in git.

Reply:
I also support dropping it. In my opinion the "author" field serves no valuable purpose.

Reply:
Agreed I'll drop it.